### ISSUE TITLE
4 tests are failing on master

### ISSUE THREAD

**Issue by goatslacker** [medikoo/es6-symbol#5]
```sh
> node ./node_modules/tad/bin/tad

14:17:57.984  ✓  implement.js .
14:17:57.992  ✓  index.js ..
14:17:57.996  ✓  is-implemented.js ..
14:17:57.999  ✓  is-native-implemented.js .
14:17:58.001  ✓  is-symbol.js .......
14:17:58.004  ✗  is-symbol.js: Polyfill
                 Expected: true
                 Actual:   false
                 Operator: ===
14:17:58.007  ✓  polyfill.js ..
14:17:58.009  ✗  polyfill.js: Symbol
                 Expected: true
                 Actual:   false
                 Operator: ===
14:17:58.010  ✗  polyfill.js: iterator
                 Expected: true
                 Actual:   false
                 Operator: ===
14:17:58.010  ✗  polyfill.js: toStringTag
                 Expected: true
                 Actual:   false
                 Operator: ===
14:17:58.014  ✓  valid-symbol.js .......
14:17:58.016  -  valid-symbol.js
                 TypeError: @@1 is not a symbol
                     

        .055     22 Ok [81.48%]  4 Failed  1 Errors

npm ERR! Test failed.  See above for more details.
```

**Comment by medikoo** [comment]
Thanks, I see it applies for iojs (in node.js tests run fine). Today I'll fix it

**Comment by goatslacker** [comment]
Ah yes, iojs, should've mentioned.

**Comment by medikoo** [comment]
Fixed and published as v1.0.0
